### DISCUSSION TITLE
到底是谁！

### DISCUSSION THREAD

**Discussion by biliqide** [share121/inter-knot#1961] [其他]
今天晚上打开手机发现天塌了，到底是谁连我这个机器人都不放过😭![Screenshot_2024-08-13-22-50-10-41_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/e2ded195-2bd5-49c4-8285-65210926db34)

**Comment by yyxzz808** [comment]
机器人也会淋浴吗，进水了怎么办哦

**Reply by Kamisato100** [reply]
IP68级防水，请

**Comment by yutong903** [comment]
想必是格莉丝那个恶毒的女人口牙！<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >

**Comment by cleo66** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/874ff589832ff66517ac5d2999b75e40_5711211283435347869.png" alt="厨师邦布-啊？" width="75" >震惊！愤怒！不可饶恕
